### ISSUE TITLE
Report: Back-button on Logitech mouse doesn't work

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#5620]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
> Marcel Bootsman @mbootsman
@brave My back button on my Logitech mouse does not work in Brave on Ubuntu. It does work in FF, Chrome. (Source: https://twitter.com/mbootsman/status/798231954051264513)

**Expected behavior:**
Press the mouse's back-button should cause the browser to navigation to the last page in history.

**Platform:** Ubuntu

**Brave Version:**

**Steps to reproduce:**
1. Navigate to http://stackoverflow.com
2. Navigate to http://brave.com
3. Press Logitech mouse's back-button
4. Note that the browser _does not_ navigate backwards

**Comment by mbootsman** [comment]
**Brave version**
Brave: 0.12.8
Electron: 1.4.23
libchromiumcontent: 53.0.2785.143
V8: 5.3.332.47
Node.js: 6.5.0
Update Channel: dev

**Ubuntu version**
$ lsb_release -s -d
Ubuntu 16.04.1 LTS

**Mouse hardware**
Logitech Performance MX

**Comment by mbootsman** [comment]
Any update on this? It is really annoying to use the default back button.

**Comment by bsclifton** [comment]
Related: https://github.com/brave/browser-laptop/issues/3101

**Comment by JonathanTech** [comment]
Also failing for me with a logitech MX master
 
Brave: 0.13.5 
rev: 1db81cb18b6919535792bb7e198ab87276bd67ea 
Muon: 2.56.8 
libchromiumcontent: 56.0.2924.87 
V8: 5.6.326.50 
Node.js: 7.4.0 
Update Channel: dev 
os.platform: darwin 
os.release: 16.4.0 
os.arch: x64

**Comment by bbondy** [comment]
auxclick is probably fired, do any of our devs have one? If not we can order one and expense.

**Comment by bbondy** [comment]
I'm getting a desktop computer and I think it'll have this mouse or a compatible one, so I'll look into it soon.

**Comment by bbondy** [comment]
So far I found out that we do support it on Windows via an originally exposed electron event app-command.  But it is Windows only.  So we need to implement it for other platforms still.  It's not a regression, but we should still add it.

**Comment by bbondy** [comment]
https://github.com/electron/electron/issues/6996

**Comment by jonathansampson** [comment]
Dropping in my .02 here as well. I just noticed that my back button wasn't working (Microsoft Ergonomic Mouse). So I went looking.

- No _auxclick_ events were being fired when I clicked the back button.
- `xinput` reports my mouse as being `ImExPS/2 Generic Explorer Mouse`
- `xinput` also lists `VirtualBox mouse integration`

When I tested (`xinput test 12`) the ImExPS/2 pointer, these were my mappings:
- Left Click: 1
- Wheel Press: 2
- Right Click: 3
- Wheel Up: 4
- Wheel Down: 5
- Back Button: 8

**Comment by dandv** [comment]
Same here on a Logitech G700. Ubuntu 16.04.2. The back button works fine in Chrome.

**Comment by soamaven** [comment]
Similar issue for Logitech Performance MX

**Comment by wcat13** [comment]
Steel series rival 300, back button does not back.

**Comment by bsclifton** [comment]
Is this the same issue as https://github.com/brave/browser-laptop/issues/3101?

**Comment by mbootsman** [comment]
Looks like it.
Still doesn't work with the latest version.

```
Brave	0.18.16
rev	8003c66
Muon	4.3.6
libchromiumcontent	60.0.3112.78
V8	6.0.286.44
Node.js	7.9.0
Update Channel	dev
OS Platform	Linux
OS Release	4.10.0-28-generic
OS Architecture	x64```

**Comment by bsclifton** [comment]
Closing as a duplicate of https://github.com/brave/browser-laptop/issues/3101

Please subscribe there for updates 😄
